ui/panels/form-settings.php

```diff
@@ -116,17 +116,18 @@
 	</div>
 
     <div class="caldera-config-group">
-        <label id="label-cf-delete-all-entries" for="cf-delete-entries-field">
+        <label id="caldera-forms-label-delete-all-entries" for="caldera-forms-delete-entries-field">
             <?php esc_html_e( 'Delete all entries', 'caldera-forms' ); ?>
         </label>
-        <div class="caldera-config-field">
-            <button id="cf-delete-all-form-entries" class="button"><?php _e('Delete all entries', 'caldera-forms'); ?></button>
-            <div id="cf-confirm-delete-all-form-entries" style="display: none;">
+        <div id="caldera-forms-delete-entries-field" class="caldera-config-field">
+            <button id="caldera-forms-delete-all-form-entries" class="button"><?php _e('Delete all entries', 'caldera-forms'); ?></button>
```

Comment:
@New0 This button is described by `#caldera-forms-delete-entries-description` correct? If so, it should have `aria-describedby` attribute.